MySQLOO/source/LuaObjectBase.cpp

```diff
@@ -153,16 +153,25 @@ void LuaObjectBase::runFunction(GarrysMod::Lua::ILuaBase* LUA, int funcRef, cons
 	va_end(arguments);
 }
 
+static int BuildErrorStack(lua_State *state) {
```

Comment:
please rename to buildErrorStack (lowercase b), to be consistent with other static function names